.coveragerc

```diff
@@ -2,7 +2,6 @@
 branch=True
 source=trio
 omit=
-  setup.py
```

Comment:
Do we want to keep a separate .coveragerc instead of putting the config to `pyproject.toml`?

Reply:
moving all config files into pyproject.toml proved controversial in gitter - I'd like to move that config in a followup PR

Reply:
@agronholm https://github.com/python-trio/trio/pull/2867